src/093 - CDMyNetwork/C-DMyNetwork/cdeMyNetwork.cs

```diff
@@ -0,0 +1,209 @@
+﻿// SPDX-FileCopyrightText: 2009-2020 TRUMPF Laser GmbH, authors: C-Labs
+//
+// SPDX-License-Identifier: MPL-2.0
+
+using System;
+using System.Collections.Generic;
+
+using nsCDEngine.Engines;
+using nsCDEngine.BaseClasses;
+using nsCDEngine.ViewModels;
+using nsCDEngine.Engines.NMIService;
+using nsCDEngine.Engines.ThingService;
+
+using CDMyNetwork.ViewModel;
+
+namespace CDMyNetwork
+{
+    public class eNetworkServiceTypes : TheDeviceTypeEnum
+    {
+        public const string PingService = "Ping Service";
+    }
+
+    partial class MyNetworkServices : ThePluginBase
+    {
+        #region ICDEPlugin
+        /// <summary>
+        /// This constructor is called by The C-DEngine during initialization in order to register this service
+        /// </summary>
+        /// <param name="pBase">The C-DEngine is creating a Host for this engine and hands it to the Plugin Service</param>
+        public override void InitEngineAssets(IBaseEngine pBase)
```

Comment:
Should we use attributes here?

Reply:
next update...when the telnet device type is coming back after cdeTelnet shared project is ready with nuget